pyaedt/mechanical.py

```diff
@@ -615,3 +616,59 @@ def assign_heat_generation(self, objects_list, value, boundary_name=""):
             self.boundaries.append(bound)
             return bound
         return False
+
+    @pyaedt_function_handler()
+    def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
+        """Create a new analysis setup for Mechanical.
+        Optional arguments are passed along with ``setuptype`` and ``setupname``.  Keyword
+        names correspond to the ``setuptype``
+        corresponding to the native AEDT API.  The list of
+        keyword here is not exhaustive.
```

Comment:
```suggestion
        """Create an analysis setup for Mechanical.
        
        Optional arguments are passed along with ``setuptype`` and ``setupname``.  Keyword
        names correspond to the ``setuptype``
        corresponding to the native AEDT API.  The list of
        keywords here is not exhaustive.
```
I think I missed earlier that this text was part of the main description. Because we should be keeping the
description short, a line break should separate this text from the short descrption.